test/dymaptic.GeoBlazor.Core.Test.Blazor.Shared/Components/SpatialReferenceTests.razor

```diff
@@ -0,0 +1,58 @@
+﻿@inherits TestRunnerBase
+
+@inject Projection projection
+
+@{
+    base.BuildRenderTree(__builder);
+}
```

Comment:
You can move this code into a .cs file and remove the `renderHandler` parameters since you're not instantiating a mapview. See `GeometryEngineTests.cs` for example.